designer/app.py

```diff
@@ -760,12 +763,10 @@ def _perform_new(self, *args):
 
         self.close_popup()
 
-        new_proj_dir = os.path.join(get_config_dir(),
-                                    constants.NEW_PROJECT_DIR_NAME)
-        if os.path.exists(new_proj_dir):
-            shutil.rmtree(new_proj_dir)
+        new_proj_dir = mkdtemp(prefix=constants.NEW_PROJECT_DIR_NAME_PREFIX,
+                               dir=get_config_dir())
```

Comment:
it's better to let the system choose the dir location.
* we are deleting these files on exit, this should work well. However, if the user kill KD, we'll have the OS to clean these folders later if using the default temp dir.